docs/source/monitoring.md

```diff
@@ -4,22 +4,36 @@ Both Prometheus, Grafana and AlertManager were configured with specific rules fo
 All of them will be available under the `scylla-monitoring` namespace.
 Customization can be done in `examples/common/monitoring/values.yaml`
 
-1. Add monitoring stack charts repository
+1. Download Scylla Monitoring
+
+   First you need to download Scylla Monitoring, which contains Grafana dashboards and custom Prometheus rules.
+   You can do this by running the following command:
+    ```
+    wget https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-4.0.0.tar.gz
+    tar -xvf scylla-monitoring-4.0.0.tar.gz
+    ```
+
+2. Add monitoring stack charts repository
    ```
    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
    helm repo update
    ```
-1. Install monitoring stack
+
+3. Install monitoring stack
     ```
-    helm install monitoring prometheus-community/kube-prometheus-stack --values examples/common/monitoring/values.yaml --create-namespace --namespace scylla-monitoring
+    yq ea '{(filename | sub(".*\/(.*)", "${1}")): .} | . as $item ireduce ({}; . * $item) | {"additionalPrometheusRulesMap": .}' scylla-monitoring-scylla-monitoring-4.0.0/prometheus/prom_rules/*.yml | helm install monitoring prometheus-community/kube-prometheus-stack --create-namespace --namespace scylla-monitoring -f examples/common/monitoring/values.yaml -f -
```

Comment:
`sub(".*\/(.*)"` -> `sub(".*\/(.+)"`

Reply:
but this won't be future proof to extra dirs, `find` with may be better for pre-processing.
with pushd (or subshell and cd) you could avoid parsing to the filename

Reply:
Question is whether we necessarily want to include subdirs? There's a subdir now `back_fill/3.8/` in `prom_rules` that contains a file whose rules intersect with one of the files from the parent dir. Also, I think creating something that works for now is good enough and there's no point in trying to proactively support future releases - at least not without any input from the monitoring team.

Reply:
i meant just `find . -type f` so it behaves the same way with a less chance to break